SteamKit2/SteamKit2/Steam/CMClient.cs

```diff
@@ -194,6 +199,7 @@ public void Send( IClientMsg msg )
 
             DebugLog.WriteLine( "CMClient", "Sent -> EMsg: {0} (Proto: {1})", msg.MsgType, msg.IsProto );
 
+            DebugNetworkListener?.OnOutgoingNetMessage( msg.MsgType, msg.Serialize() );
```

Comment:
What happens if the debug listener throws an exception? Same for incoming below.